src/Replication.php

```diff
@@ -524,7 +527,7 @@ public function replicateChanges(array &$revDiff)
             // transferred revision that had attachment in the current doc.
             $allResponse['multipartResponse'][$docId] = $multipartResponse;
         }
-        $allResponse['bulkResponse'] = $bulkUpdater->execute()->body;
+        $allResponse['bulkResponse'] = $bulkUpdater->executeByLimit(100);
```

Comment:
Maybe this limit should be configurable too but default to 100? or is there a reason it is not configurable? The other `$limit` property for changes seems ambiguous - maybe it should have a more specific name, like `$changesLimit`. Then this could be `$bulkDocsLimit` or something.